releasenotes/notes/display-configure-errors-status-page-dff9f92a97e5a95e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    The status page now displays errors raised by the '__init__' method of a
```

Comment:
I think this should say the "The status **command**..."

Reply:
We also have the GUI who show the same thing (I'll add both).